magento2/usr/local/share/magento2/magento_functions.sh

```diff
@@ -408,10 +408,15 @@ function do_magento_download_magerun2() {
   chmod +x /app/bin/n98-magerun2.phar
 }
 
-function do_magento2_templating() {
-  if [ "$MAGENTO_VERSION" != "2.0" ] && [ "$MAGENTO_VERSION" != "2.1" ]; then
+function remove_config_template() (
+  set +e
```

Comment:
you don't need to `set +e` or sub-shell. the `if` condition keeps the script running

Reply:
for instance, how we did `[ "$MAGENTO_VERSION" != "2.0" ]`, this would return an exit status 0/1 based on if it matches. Its the `if` that catches it instead of the -e flag causing it to stop further execution and show the error